apprentice_teacher.rb

```diff
@@ -1,53 +1,23 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'Teacher'
+
+class ApprenticeTeacher < Teacher
+  include Teachable
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
+    @benchmark_rating = 80
   end
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
+  def attend_training_session
+    puts "Whoa. I know ruby-fu"
   end
 
-  def teach_stuff
+    def teach_stuff
```

Comment:
Don't introduce bad indentation! Indentation may seem trivial, as the program works whether you're following standard conventions or not.  But when you're working on a large project with lots of other people, you want to know that everyone is following the same style guide.  Even just looking back on your own code, you'll prefer well formatted code over poorly formatted code!  And any workplace with code reviews in place will require that you follow their style guide, so get used to it now.